pandas/io/tests/test_excel.py

```diff
@@ -543,6 +506,24 @@ def test_read_excel_blank_with_header(self):
         actual = read_excel(blank, 'Sheet1')
```

Comment:
These tests above `test_read_excel_blank` and `test_read_excel_blank_with_header` should be changed to the new structure.